### DISCUSSION TITLE
UM Documentation and Useful Links

### DISCUSSION THREAD

**Discussion by richardgilham** [MetOffice/simulation-systems#66] [UM]
**Working Practices:** https://metoffice.github.io/simulation-systems/

**TRAC:** https://code.metoffice.gov.uk/trac/um/wiki

Please drop any suggestions for further links as a comment as we can edit this pinned post.

**Comment by soi521** [comment]
How about adding these? UM Training https://code.metoffice.gov.uk/doc/um/latest/um-training/index.html and the latest pdf version of the UM manual (June 2022): https://code.metoffice.gov.uk/doc/um/latest/papers/umdp_000.pdf
